### PR TITLE
Use HTML email for 2FA

### PR DESCRIPTION
**Pull request by j0k3r**
Related to #1490

### CODE REVIEW

**Comment by nicosomb** [comment]
@j0k3r what is your TODO list on this PR? Are you waiting for @modos189?

**Comment by j0k3r** [comment]
I'm just waiting for the html.

**Comment by j0k3r** [comment]
Preview: 

image

File: src/Wallabag/UserBundle/Resources/views/TwoFactor/email_auth_code.html.twig [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+    <table cellpadding="0" cellspacing="0" border="0" id="backgroundTable">
+        <tr>
+            <td>
+
+                <table cellpadding="0" cellspacing="0" border="0" align="center" id="card">
+                    <tr>
+                        <td style="padding: 20px;" width="96px" valign="top"><img class="image_fix" src="{ wallabag_url }/themes/material/img/logo-other_themes.png" alt="logo" title="{ wallabag_url }" style="width: 96px; height: 96px;" /></td>
+                        <td style="padding: 20px; padding-left: 0;" valign="top" id="cell_desc">
+                            <h1>Wallabag</h1>
```
Comment (nicosomb) [inline src/Wallabag/UserBundle/Resources/views/TwoFactor/email_auth_code.html.twig]: wallabag ;-)

File: src/Wallabag/UserBundle/Resources/translations/wallabag_user.en.yml [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
-auth_code.mailer.body: |
-    Hi %user%,
-
+auth_code.mailer.body.hello: "Hi %user%,"
+auth_code.mailer.body.content: |
     Since you enable two factor authentication on your wallabag account and you just logged in from a new device (computer, phone, etc.), we send you a code to validate your connection.
     Here is the code: %code%
-
+auth_code.mailer.body.signature: |
     Please don't hesitate to contact us if you have any problems: %support%
```
Comment (nicosomb) [inline src/Wallabag/UserBundle/Resources/translations/wallabag_user.en.yml]: The new line is not visible on your screenshot.

Comment (modos189) [inline src/Wallabag/UserBundle/Resources/translations/wallabag_user.en.yml]: html tags is not supported in translations =(

Comment (j0k3r) [inline src/Wallabag/UserBundle/Resources/translations/wallabag_user.en.yml]: Yeah, I'll split them

**Comment by j0k3r** [comment]
Update preview, html:
image

text:
image

**Comment by modos189** [comment]
Perfect)

File: src/Wallabag/UserBundle/Resources/views/TwoFactor/email_auth_code.html.twig [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+                        </td>
+                    </tr>
+                    <tr><td colspan="2" style="padding: 0;"><div style="height: 0; border-top: 1px solid #c5c5c5;">&nbsp;</div></td></tr>
+                    <tr>
+                        <td colspan="2">
+
+                            <table cellpadding="0" cellspacing="0" border="0" width="100%">
+                                <tr>
+                                    <td valign="top" style="padding: 20px; text-align: center"><a href="{{ wallabag_url }}">{{ wallabag_url }}</a></td>
+                                    <td valign="top" style="padding: 20px; text-align: center">Powered by <a href="https://www.wallabag.org/">Wallabag</a></td>
```
Comment (nicosomb) [inline src/Wallabag/UserBundle/Resources/views/TwoFactor/email_auth_code.html.twig]: `wallabag` (sorry, I didn't see it yesterday)

Comment (modos189) [inline src/Wallabag/UserBundle/Resources/views/TwoFactor/email_auth_code.html.twig]: Why wallabag, but no Wallabag? :D

Comment (nicosomb) [inline src/Wallabag/UserBundle/Resources/views/TwoFactor/email_auth_code.html.twig]: http://support.wallabag.org/kb/faq/wallabag-or-wallabag

**Pull request merged**